ooni/nettest.py

```diff
@@ -219,7 +220,8 @@ def testDetails(self):
             'input_hashes': input_file_hashes,
             'report_id': self.reportID,
             'test_helpers': self.testHelpers,
-            'annotations': self.annotations
+            'annotations': self.annotations,
+            'data_format_version': '0.2.0'
```

Comment:
This was added in order to be able to distinguish inside of the data pipeline when a report to be parsed is using the new data format (and hence doesn't require being converted) or is using the old data format.

The ones that end up being published in the end will all have this attribute set to 0.2.0, but not when the are inserted first in the pipeline.